pkgs/applications/science/electronics/openhantek6022/cmake-paths.patch

```diff
@@ -0,0 +1,51 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 192823e..c66aa7b 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -59,36 +59,36 @@ if (WIN32)
+ endif()
+ 
+ if("${CMAKE_SYSTEM}" MATCHES "Linux")
+-  if(EXISTS "/lib/udev/rules.d/")
```

Comment:
My other comment about `substituteInPlace` probably also applies here. (And/or use `sed`.)